app/views/components/fields/field_text.scala.html

```diff
@@ -18,7 +18,7 @@
 
 @(
     field: Field,
-    label: String,
+    label: Option[String] = None,
```

Comment:
@PauloPortugal what do you think abut use here empty string as null object pattern ?

Reply:
I agree with @bambuchaAdm . In 90% we will have some value here and this component handle empty string correctly.

Reply:
We have decided to go with `Option` because the component signature has all the other text elements as optional as well.
We probably need to review this with the design/product team and understand if this will be the new way of creating text fields, i.e., having a section header, `h1` and a header page hint. 
If so we will have to create a new component and probably then we would avoid optional fields and use just Strings.